README.md

```diff
@@ -127,11 +129,10 @@ This plugin uses the same router as the `fastify` instance, this has a few impli
 - When using `@fastify/websocket`, it needs to be registered before all routes in order to be able to intercept websocket connections to existing routes and close the connection on non-websocket routes.
 
 ```js
-'use strict'
-
-const fastify = require('fastify')()
+import Fastify from 'fastify'
 
-fastify.register(require('@fastify/websocket'))
+const fastify = Fastify()
+await fastify.register(require('@fastify/websocket'))
```

Comment:
Maybe more consistent to use an es import for this instead of mixing styles?

Reply:
I did the best of showing both stiles, either with with `await app.register()` or with using a plugin.